mautic/mautic
# Install At.js via npm

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [cleanup]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes [TPROD-434](https://mautic.atlassian.net/browse/TPROD-434) 



#### Description:
Installing At.js via npm package manager. There were one hack for At.js, so created patch file for at.js.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. In Create Email > "Advance" tab and type `{` in Plain Text Version, the token list should be displayed.
3. Disable `GrapesJs` plugin
4. Create new email
5. Select the Mauve theme
6. Open the builder & click on some text slot
8. In the right hand column you'll see a WYSIWYG editor for that text slot
9. Type `{` and you should see the list of tokens as per the following screenshot.
    
<img width="300" alt="image" src="https://github.com/mautic/mautic/assets/68939488/6b588165-7163-4e9b-acae-a30503810796">





[TPROD-434]: https://mautic.atlassian.net/browse/TPROD-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
